apps/tests/integration/integration_test.go

```diff
@@ -23,16 +25,18 @@ import (
 )
 
 const (
-	msIDlabDefaultScope = "https://msidlab.com/.default"
-	graphDefaultScope   = "https://graph.windows.net/.default"
-)
-
-const microsoftAuthorityHost = "https://login.microsoftonline.com/"
+	// URLS
+	msIDlabDefaultScope    = "https://request.msidlab.com/.default"
+	graphDefaultScope      = "https://graph.windows.net/.default"
+	microsoftAuthorityHost = "https://login.microsoftonline.com/"
 
-const (
 	organizationsAuthority = microsoftAuthorityHost + "organizations/"
-	microsoftAuthority     = microsoftAuthorityHost + "microsoft.onmicrosoft.com"
+	microsoftAuthority     = microsoftAuthorityHost + "72f988bf-86f1-41af-91ab-2d7cd011db47"
 	//msIDlabTenantAuthority = microsoftAuthorityHost + "msidlab4.onmicrosoft.com" - Will be needed in the future
+
+	// Default values
+	defaultClientId = "f62c5ae3-bf3a-4af5-afa8-a68b800396e9"
+	pemFile         = "../../../cert.pem"
```

Comment:
Can you please add a testing.md doc to explain how app developers can run tests on their local boxes?